st2common/st2common/openapi.yaml

```diff
@@ -2069,6 +2074,72 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  /api/v1/rbac/role_assignments:
```

Comment:
How are we handling API versioning? This seems like a substantial change; even though it's purely additive, I'm wondering when we should worry about bumping versions.